src/plugin.ts

```diff
@@ -53,6 +77,7 @@ class LiftPlugin {
 
     constructor(serverless: Serverless, cliOptions: Record<string, string>) {
         this.serverless = serverless;
+        Object.assign(this.serverless, { getProviderById: this.getProviderById.bind(this) });
```

Comment:
`getProviderById` -> `getLiftProviderById`?

To avoid any surprise/conflict, because Lift providers are not the same as the existing provider system.

Also it seems like it's added for tests only. It might be worth documenting this, e.g.

```suggestion
        // @internal This method is exposed for Lift tests only, it is not a public API
        Object.assign(this.serverless, { getProviderById: this.getProviderById.bind(this) });
```